data.yaml

```diff
@@ -1,11 +1,11 @@
 active_epochs: ["2023.7"]
 next_epoch: "2023.9"
-distros: [core, community]
+distros: [core, community, minimal]
```

Comment:
I'm apparently mildly attached to the idea of `qiime2-tiny` being a meta-package, any chance of `tiny`?

Reply:
That works.